doc/install_generic.inc

```diff
@@ -0,0 +1,102 @@
+.. include:: css.inc
+
+.. _install_generic:
+
+
+Installation
+============
+
+Stable Installation
+-------------------
+
+Install the latest stable version of Theano with:
+
+.. raw:: html
+
+    <div class="highlight"><pre><span class="red">&lt;sudo&gt;</span> pip install <span class="blue">&lt;--user&gt;</span> Theano[test, doc]</pre></div>
+
+- Any argument between <...> is optional.
+
+- Use :red:`sudo` for a root installation.
+
+- Use :blue:`user` for a user installation without admin rights. It will install Theano in your local site-packages.
+
+- [test] will install the requirements for testing.
+
+- [doc] will install the requirements in order to generate the documentation.
+
+If you encountered any trouble, head to the :ref:`troubleshooting` page.
+
+libgpuarray
+^^^^^^^^^^^
+
+For the stable version of Theano you need a specific version of libgpuarray, 
+that has been tagged ``v-9998``.
+Download it with:
+
+.. raw:: html
+
+    <div class='highlight'><pre>
+    git clone https://github.com/Theano/libgpuarray.git --tags
+    git checkout origin/v-9998
+    cd libgpuarray
+    </pre></div>
+
+and then follow the `Step-by-step instructions <http://deeplearning.net/software/libgpuarray/installation.html#step-by-step-install>`__.
+
+
+Bleeding-Edge Installation (recommended)
+----------------------------------------
+
```

Comment:
For the bleeding edge and developer installation, we should have a direct link to the online version at http://deeplearning.net/software/theano_versions/dev/ and a warning to make sure to use the latest instructions.
Otherwise, people may follow earlier, outdated versions of the instructions.